plugins/vlc-video/vlc-video-plugin.h

```diff
@@ -1,16 +1,16 @@
 #include <obs-module.h>
-#include <libvlc.h>
+#include <vlc/libvlc.h>
```

Comment:
This definitely is not necessary.  I've tested this on all three major operating systems and this has never been a problem.  In the cmake module in cmake/Modules/FindLibVLC.cmake, you'll notice it searches suffixes: vlc, include/libvlc.  This is most likely a problem with your setup rather than with the code itself.